### DISCUSSION TITLE
Showing off the music player I made with `fluent_ui`

### DISCUSSION THREAD

**Discussion by Losses** [bdlukaa/fluent_ui#1123] [Show and tell]
It is still buggy and heavily alpha, and the source code looks messy for now tho, but I believe the player looks great and is worth sharing ;)
 
https://github.com/user-attachments/assets/b4a555f2-a87e-4c16-82da-b80a62473cc5

Source: https://github.com/losses/rune

**Comment by bdlukaa** [comment]
Wow! Looks really great. Love the animations, the slider at the bottom and the adapted text field. Keep going!
